okteto.yml

```diff
@@ -0,0 +1,20 @@
+name: minio-operator
+image: okteto/golang:1
+command:
+  - bash
+workdir: /okteto
+namespace: minio-operator
+securityContext:
+  capabilities:
+    add:
+      - SYS_PTRACE
+volumes:
+  - /go/pkg/
+  - /root/.cache/go-build/
+sync:
+  - .:/okteto
+forward:
+  - 2345:2345
+  - 8080:8080
+persistentVolume:
+  enabled: true
```

Comment:
Do not add this? @stefanhenseler

Reply:
I removed it and force pushed. should be clean again.